src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -164,7 +164,7 @@ public void periodic() {
         if (false == isExtenderCalibrated) {
             // The extender is not calibrated, start the extender calibration
             extenderTalonFX.set(-EXTENDER_CALIBRATION_MOTOR_SPEED);
-        } else if (false == isPivotCalibrated) {
+        } else if (!isPivotCalibrated) {
```

Comment:
Can we revert this conditional back to false == variable? More obvious than the !

Reply:
Done